README.md

```diff
@@ -36,6 +36,33 @@ You can build your own or choose from the community released ones:
 You can try a Volto online demo at [https://demo.plone.org/](https://demo.plone.org/).
 
 
+## Monorepo structure
+
+Since version 18.0.0-alpha4, the Volto core repository has had the shape of a monorepo, where "mono" means "single" and "repo" is short for "repository".
+This means that several apps and libraries related to each other are stored in the same repository.
+They are managed together but released individually.
+This allows the code to be shared effectively, and unifies tracking of changes across all of the apps and libraries.
+
+Some packages released to npm were previously in separate repositories, but are now included in this monorepo.
+They include the following items.
```

Comment:
I think all of these were first developed in this repository. I would just remove this paragraph.